src/views/FieldReportForm/ResponseFields/styles.module.css

```diff
@@ -0,0 +1,2 @@
+.contacts-section {
+}
```

Comment:
1. Let's not leave it empty
2. 
```suggestion
.response-fields {
}
```

Reply:
Styling is a WIP